src/txn/txn_ckpt.c

```diff
@@ -1238,6 +1240,25 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
     WT_WITH_SCHEMA_LOCK(session, ret = __checkpoint_prepare(session, &tracking, cfg));
     WT_ERR(ret);
 
+    WT_ERR(__wt_config_gets(session, cfg, "debug.checkpoint_crash_point", &cval));
+    ckpt_crash_random = (int)cval.val;
```

Comment:
How about `ckpt_crash_point`?

Reply:
@kommiharibabu To align with the existing debug checkpoint configuration variable `checkpoint_cleanup`, I've named the new variable `checkpoint_crash_point`.